apps/emqx_durable_storage/src/emqx_ds_storage_reference.erl

```diff
@@ -115,7 +115,7 @@ commit_batch(_ShardId, #s{db = DB, cf = CF}, Messages) ->
         end,
         Messages
     ),
-    Res = rocksdb:write_batch(DB, Batch, _WriteOptions = []),
```

Comment:
This should be passed as a parameter to the storage layer. Replication-less edition will use rocksdb WAL.